config/initializers/string.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class String
+  def to_title
```

Comment:
If we are going to monkey-patch this, could we add some info to make it clear why we've done this?
I would suggest a comment here and maybe make it clear in the method name, something like `titleize_with_hyphens` or `titleize_preserve_hyphens`

Reply:
Fixed